includes/class-windows-azure-helper.php

```diff
@@ -512,7 +512,7 @@ static public function put_media_to_blob_storage( $container_name, $blob_name, $
 	 *
 	 * @return bool|string|WP_Error False or WP_Error on failure URI on success.
 	 */
-	static public function copy_media_to_blob_storage( $container_name, $destination_path, $source_path, $mime_type, $account_name = '', $account_key = '' ) {
+	static public function copy_media_to_blob_storage( $container_name, $destination_path, $source_path, $mime_type, $account_name = '', $account_key = '', $cache = null ) {
```

Comment:
Since we're adding a new function param here, we should document that in the docblock